### DISCUSSION TITLE
Jukebox improvements

### DISCUSSION THREAD

**Discussion by happyburger23** [Roadhog360/Et-Futurum-Requiem#320] [Ideas]
- Jukeboxes should display a note particle above the block when it is being used
- Hoppers/droppers/hopper minecarts should be able to insert/remove discs from jukeboxes
- Jukeboxes should use wood sounds, not stone
- Should emit redstone signals corresponding to the disc that was inserted

**Comment by Roadhog360** [comment]
The wood sounds have been added in a recent commit.
